app/javascript/app/phone-internationalization.js

```diff
@@ -51,14 +54,14 @@ const updateOTPDeliveryMethods = () => {
     return;
   }
 
-  const phoneLabel = phoneRadio.parentNode.parentNode;
+  const phoneLabel = /** @type {Element} */ (phoneRadio.parentNode).parentNode;
   const deliveryMethodHint = document.querySelector('#otp_delivery_preference_instruction');
 
   const warningMessage = unsupportedInternationalPhoneOTPDeliveryWarningMessage();
   if (warningMessage) {
     disablePhoneState(phoneRadio, phoneLabel, smsRadio, deliveryMethodHint, warningMessage);
   } else {
-    enablePhoneState(phoneRadio, phoneLabel, smsRadio, deliveryMethodHint);
+    enablePhoneState(phoneRadio, phoneLabel, deliveryMethodHint);
```

Comment:
FYI this appears to have been previously incorrect. TypeScript caught it. `enablePhoneState` takes three arguments.

Reply:
would it have been throwing JS errors when used?

Reply:
> would it have been throwing JS errors when used?

Just by looking: It probably wouldn't throw, since the `smsRadio` we were passing as third argument is an `HTMLInputElement`, for which [the `innerText` assignment](https://github.com/18F/identity-idp/blob/a90670eddfa35e9ded5c63466cb62b3c505e02e9/app/javascript/app/phone-internationalization.js#L37-L39) is valid. It probably doesn't behave as we'd expect. I'll do some real-world testing to see that this behaves as we expect.

Reply:
So I think this code is never being reached, due to a bug with how we're populating the country code data.

The warning message is generated if there's a `data-sms-only` attribute:

https://github.com/18F/identity-idp/blob/f5188069aca3f2028cf0db4d1864d29a0b70b767/app/javascript/app/phone-internationalization.js#L10-L19

We pass the data attributes via:

https://github.com/18F/identity-idp/blob/f5188069aca3f2028cf0db4d1864d29a0b70b767/app/helpers/form_helper.rb#L99-L105

Which pulls from this YAML data:

https://github.com/18F/identity-idp/blob/f5188069aca3f2028cf0db4d1864d29a0b70b767/config/country_dialing_codes.yml#L4-L8

`sms_only` isn't a property. Looks like this was changed in #4331.

We probably want to update this logic, optionally pulling in `PhoneNumberCapabilities#sms_only?` if it makes sense.

Reply:
oh yikes... good catch re `sms_only` ..... my bad

Reply:
Re: Wrong arguments being passed: Yes, we're assigning an `innerText` to the radio input field itself, which AFAICT is a noop. The problem would manifest itself if the user would select an option which cannot receive phone calls, then switch back to one which can. Normally, this should reset the hint text. Instead, it sticks with the error text:

![image](https://user-images.githubusercontent.com/1779930/106198019-7108a700-6181-11eb-8287-bfa692554fd9.png)

(Again, due to above behavior with `sms_only`, this won't ever happen currently)